### PR TITLE
Updated readme with JVM DNS TTL Notice

### PR DESCRIPTION
**Pull request by ralgozino**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,8 @@ See the [official documentation for more details](https://hub.docker.com/r/jboss
 
 > ⚠️ If you not set an external database, keycloak will default to a `H2` local instance on every pod. This means that it will not be syncronized between the different keycloak pods.
 
+> ⚠️ On some cases, the Java Virtual Machine (JVM) is configure by default to cache DNS name lookups **forever** instead of following the record's TTL. This could be problematic in cases where Keycloak should contact endpoints that have "dynamic" DNS entries, for example AWS' RDS endpoint. In order to disable the infit cache, you need to pass the `-Dnetworkaddress.cache.ttl=60` flag to the JVM. Where `60` is the TTL in seconds you want to use. You can add it to the `JAVA_OPTS` environment variable.
```
Comment (phisco) [inline README.md]: typo: infinit => infinite.

Comment (phisco) [reply README.md]: shouldn't this be a default flag for the command?

Comment (ralgozino) [reply README.md]: I'm not sure, I thought about it and haven't made my mind yet so I decided to at least add the warning to the docs.

Should we change a default behaviour of the JVM? If you are not using using something like RDS it doesn't seem to affect much. On the other hand, could it have some other unexpected behaviour changing this default?

On the other hand, we could do the opossite, change the default and add a warning explaining why we changed it.

I would like to know your opinion.

Comment (angelbarrera92) [reply README.md]: IMHO i prefer to not alter the JVM parameters by default. This warning is enough

Comment (phisco) [reply README.md]: fair enough! let's merge it

**Pull request merged**